src/libraries/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.SecKeyRef.cs

```diff
@@ -141,7 +145,7 @@ internal static int GetSimpleKeySizeInBits(SafeSecKeyRefHandle publicKey)
                         externalRepresentation = CoreFoundation.CFGetData(data);
                         return true;
                     case kErrorSeeError:
-                        if (Interop.CoreFoundation.GetErrorCode(errorHandle) == errSecPassphraseRequired)
+                        if (Interop.CoreFoundation.GetErrorCode(errorHandle) is errSecPassphraseRequired or errSecInvalidKeyAttributeMask)
```

Comment:
We can handle the error code change here, but it might still be fragile. I guess Apple could always change it again.

We should investigate if there is a better way to determine if a key needs to be exported with the legacy API. Some attribute of the key?

Regardless, I kept the change simple for now since we likely want to backport this.